### ISSUE TITLE
backPressed not triggered

### ISSUE THREAD

**Issue by rolandoldengarm** [initialxy/cordova-plugin-themeablebrowser#78]
````
backButton: {     
    wwwImage: 'img/arrow_left_2.png',
    wwwImagePressed: 'img/arrow_left_2.png',
    wwwImageDensity: 3,
    align: 'left',
    event: 'backPressed'
}
...
cordova.ThemeableBrowser.open(url, options, themeableBrowserConfig)
        .addEventListener('backPressed', function (e) {
            alert('back pressed');
        });
````
The back button is visible, but nothing happens when I click on it.
I use wwwImage, just to test quickly.

**Comment by rolandoldengarm** [comment]
Even when I use native images (instead of wwwImage) the event is not fired.

**Comment by rolandoldengarm** [comment]
The issue relates to backButtonCanClose. If that flag is not set to true, the backPressed event is not raised....

**Comment by sencenan** [comment]
@rolandoldengarm That seems to be intentional. Is this still a problem?

**Comment by rolandoldengarm** [comment]
I think it's weird if this is intentional. The properties are related, but you may want to have a backbutton event, but not be able to close the browser with it.

**Comment by sencenan** [comment]
@rolandoldengarm Sorry I might not have read it carefully enough. Is this similar to this issue: https://github.com/initialxy/cordova-plugin-themeablebrowser/issues/77 ?

**Comment by rolandoldengarm** [comment]
Yes I guess so. But in my case the issue happened on Android as well.
